configs/predict.yaml

```diff
@@ -5,6 +5,7 @@ defaults:
   - _self_
   - dataset: conll2003
   - serializer: json
+  - paths: default.yaml
 
   # enable color logging
   - override hydra/hydra_logging: colorlog
```

Comment:
can we add `task_name: "predict"` down below to follow the scheme of train/eval?

Reply:
and also `data_dir` is already defined in `paths`

Reply:
Ah yeah right, task_name is required for logs so must be added.